src/proj.def

```diff
@@ -146,3 +146,6 @@ EXPORTS
     proj_list_ellps                @132
     proj_list_units                @133
     proj_list_prime_meridians      @134
+
+    pj_left                        @135
```

Comment:
These functions are private, right? If that's the case, they shouldn't not be here.